src/ddmd/astbase.d

```diff
@@ -0,0 +1,6303 @@
+module ddmd.astbase;
+
+import ddmd.astbasevisitor;
+
+/** The ASTBase  family defines a family of AST nodes appropriate for parsing with
```

Comment:
Ddoc will interpet the first paragraph as the summary [1]. So I recommend splitting it up to get a summary paragraph, perhaps after the first sentence.

[1] http://dlang.org/spec/ddoc.html#summary